auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -141,12 +141,16 @@
           <span>{{ status(item) }}</span>
           <!-- Future Effective Icon for PAID_FE status -->
           <template v-if="status(item) === BusinessState.PAID_FE">
-            <v-tooltip bottom>
+            <v-tooltip
+              top
+              :max-width="'300px'"
+            >
               <template #activator="{ on, attrs }">
                 <v-icon
                   class="ml-1"
                   color="#F8661A"
                   v-bind="attrs"
+                  style="cursor: pointer; font-size: 1.5em; margin-left: 4px;"
```

Comment:
Inline styles should be avoided.

If you can't use the IconTooltip sub-component here then please create a class and move these styles to the CSS block.

Reply:
Updated to use the IconTooltip sub-component